### PR TITLE
[Keyboard] Add Axolstudio Yeti

### PR DESCRIPTION
**Pull request by kb-elmo**
## Description

Added Axolstudio Yeti keyboard. VIA keymap included.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by zvecr — CHANGES_REQUESTED** [review]
qmk info doesnt seem to like your `Layout_Splitbs` layout, as shown in the CI lint check
```console
┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐
│k1││k0││k0││k0││k0││k0││k0││k0││k0││k0││k0││k1││k1││k1││k1││k1││k3││k1││k1││k1││k2││k2││k2││k2││k2││k2││k2││k2││k2││k2││k3││k4││k3││k3││k3││k3││k3││k3││k3││k4││k4││k4││k4││k4││k3││k4││k5││k5││k5││k5││k5││k5││k5││k5││k5││k5││k6││k4││k6││k6││k6││k6││k6││k6││k6│
└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘
```

Any chance you could have a look at fixing this.

**Comment by kb-elmo** [comment]
interesting. I'll see if I find anything

File: keyboards/axolstudio/yeti/yeti.h [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+         k61,      k62,      k63, k64,      k65,      k66,                     k67  \
+) { \
+    { k00,   k01,   k02,   k03,   k04,   k05,   k06,   k07,   k08,   k09,   k10,   k11,   k12,   k13, KC_NO,   k15 }, \
+    { k16,   k17,   k18,   k19,   k20,   k21,   k22,   k23,   k24,   k25,   k26,   k27,   k28,   k29,   k30,   k31 }, \
+    { k32,   k33,   k34,   k35,   k36,   k37,   k38,   k39,   k40,   k41,   k42,   k43,   k44,   k45, KC_NO,   k47 }, \
+    { k48,   k49,   k50,   k51,   k52,   k53,   k54,   k55,   k56,   k57,   k58,   k59,   k60, KC_NO, KC_NO, KC_NO }, \
+    { KC_NO, k61, KC_NO,   k62, KC_NO,   k63,   k64, KC_NO,   k65, KC_NO,   k66, KC_NO, KC_NO, KC_NO, KC_NO,   k67 }  \
+}
+
+#define LAYOUT_splitbs( \
```
Comment (zvecr) [inline keyboards/axolstudio/yeti/yeti.h:49]: This should fix the previously mentioned CI issue

```suggestion
#define LAYOUT_split_bs( \
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/axolstudio/yeti/rules.mk [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = no        # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = no            # USB Nkey Rollover
```
Comment (fauxpark) [inline keyboards/axolstudio/yeti/rules.mk:15]:
```suggestion
BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = no            # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/axolstudio/yeti/rules.mk (hunk 2 above)
Comment (kb-elmo) [reply keyboards/axolstudio/yeti/rules.mk:15]: why add options like RGBLIGHT, BACKLIGHT or BLUETOOTH when the PCB has absolutely no support for it?

File: keyboards/axolstudio/yeti/config.h [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID    0x05E4
```
Comment (drashna) [inline keyboards/axolstudio/yeti/config.h:23]: This needs to be changed, as the VID is in use. 
https://www.the-sz.com/products/usbid/index.php?v=0x05E4&p=&n=

```
❯ echo -n "Axolstudio" | shasum | cut -c1-4 | tr '[:lower:]' '[:upper:]'
525C
```

```suggestion
#define VENDOR_ID    0x525C
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/axolstudio/yeti/keymaps/via/keymap.c [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+                 KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT,            MO(1),
+                 KC_LCTL,          KC_LALT,          KC_SPC,  KC_LGUI,          KC_SPC,           KC_RALT,                                    KC_RCTL
+    ),
+    [1] = LAYOUT_all(
+        KC_TRNS, KC_GRV,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_HOME, KC_UP,   KC_END,  KC_PGUP, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,
+        KC_TRNS, KC_CAPS, KC_LEFT, KC_DOWN, KC_RGHT, KC_PGDN, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,
+                 KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_VOLD, KC_VOLU, KC_MUTE, KC_MPRV, KC_MNXT, KC_MPLY, KC_TRNS,          KC_TRNS,
+                 KC_TRNS,          KC_TRNS,          KC_TRNS, KC_TRNS,          KC_TRNS,          KC_TRNS,                                     KC_TRNS
+    )
```
Comment (fauxpark) [inline keyboards/axolstudio/yeti/keymaps/via/keymap.c:33]: Needs two more empty layers.

**Review by fauxpark — APPROVED** [review]

File: keyboards/axolstudio/yeti/keymaps/via/keymap.c (hunk 4 above)
Comment (kb-elmo) [reply keyboards/axolstudio/yeti/keymaps/via/keymap.c:47]: completely forgot those. added them :)

**Review by drashna — APPROVED** [review]

File: keyboards/axolstudio/yeti/yeti.h [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT( \
```
Comment (noroadsleft) [inline keyboards/axolstudio/yeti/yeti.h:21]:
```suggestion
#define LAYOUT_alice( \
```

File: keyboards/axolstudio/yeti/yeti.h [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+         k61,      k62,      k63, k64,      k65,      k66,                     k67  \
+) { \
+    { k00,   k01,   k02,   k03,   k04,   k05,   k06,   k07,   k08,   k09,   k10,   k11,   k12, KC_NO,   k14, KC_NO }, \
+    { k16,   k17,   k18,   k19,   k20,   k21,   k22,   k23,   k24,   k25,   k26,   k27,   k28,   k29,   k30,   k31 }, \
+    { k32,   k33,   k34,   k35,   k36,   k37,   k38,   k39,   k40,   k41,   k42,   k43,   k44,   k45, KC_NO,   k47 }, \
+    { k48,   k49,   k50,   k51,   k52,   k53,   k54,   k55,   k56,   k57,   k58,   k59,   k60, KC_NO, KC_NO, KC_NO }, \
+    { KC_NO, k61, KC_NO,   k62, KC_NO,   k63,   k64, KC_NO,   k65, KC_NO,   k66, KC_NO, KC_NO, KC_NO, KC_NO,   k67 }  \
+}
+
+#define LAYOUT_all( \
```
Comment (noroadsleft) [inline keyboards/axolstudio/yeti/yeti.h:35]:
```suggestion
// Equivalent to LAYOUT_all
#define LAYOUT_alice_split_bs( \
```

File: keyboards/axolstudio/yeti/yeti.h [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+         k61,      k62,      k63, k64,      k65,      k66,                     k67  \
+) { \
+    { k00,   k01,   k02,   k03,   k04,   k05,   k06,   k07,   k08,   k09,   k10,   k11,   k12,   k13, KC_NO,   k15 }, \
+    { k16,   k17,   k18,   k19,   k20,   k21,   k22,   k23,   k24,   k25,   k26,   k27,   k28,   k29,   k30,   k31 }, \
+    { k32,   k33,   k34,   k35,   k36,   k37,   k38,   k39,   k40,   k41,   k42,   k43,   k44,   k45, KC_NO,   k47 }, \
+    { k48,   k49,   k50,   k51,   k52,   k53,   k54,   k55,   k56,   k57,   k58,   k59,   k60, KC_NO, KC_NO, KC_NO }, \
+    { KC_NO, k61, KC_NO,   k62, KC_NO,   k63,   k64, KC_NO,   k65, KC_NO,   k66, KC_NO, KC_NO, KC_NO, KC_NO,   k67 }  \
+}
+
+#define LAYOUT_split_bs( \
```
Comment (noroadsleft) [inline keyboards/axolstudio/yeti/yeti.h:49]:
```suggestion
#define LAYOUT_alice_split_bs_full_rshift( \
```

File: keyboards/axolstudio/yeti/yeti.h [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+         k61,      k62,      k63, k64,      k65,      k66,                     k67  \
+) { \
+    { k00,   k01,   k02,   k03,   k04,   k05,   k06,   k07,   k08,   k09,   k10,   k11,   k12,   k13, KC_NO,   k15 }, \
+    { k16,   k17,   k18,   k19,   k20,   k21,   k22,   k23,   k24,   k25,   k26,   k27,   k28,   k29,   k30,   k31 }, \
+    { k32,   k33,   k34,   k35,   k36,   k37,   k38,   k39,   k40,   k41,   k42,   k43,   k44, KC_NO,   k46, KC_NO }, \
+    { k48,   k49,   k50,   k51,   k52,   k53,   k54,   k55,   k56,   k57,   k58,   k59,   k60, KC_NO, KC_NO, KC_NO }, \
+    { KC_NO, k61, KC_NO,   k62, KC_NO,   k63,   k64, KC_NO,   k65, KC_NO,   k66, KC_NO, KC_NO, KC_NO, KC_NO,   k67 }  \
+}
+
+#define LAYOUT_full_rshift( \
```
Comment (noroadsleft) [inline keyboards/axolstudio/yeti/yeti.h:63]:
```suggestion
#define LAYOUT_alice_full_rshift( \
```

File: keyboards/axolstudio/yeti/info.json [hunk 9]
```diff
@@ -0,0 +1,285 @@
+{
+  "keyboard_name": "yeti",
+  "url": "https://axolstudio.ca/yeti",
+  "maintainer": "kb-elmo",
+  "width": 17.75,
+  "height": 5,
+  "layouts": {
+    "LAYOUT": {
```
Comment (noroadsleft) [inline keyboards/axolstudio/yeti/info.json:8]:
```suggestion
    "LAYOUT_alice": {
```

File: keyboards/axolstudio/yeti/info.json [hunk 10]
```diff
@@ -0,0 +1,285 @@
...
+        {"x":1.25, "y":4, "w":1.5},
+        {"x":4.25, "y":4, "w":1.5},
+        {"x":5.75, "y":4, "w":2},
+        {"x":7.75, "y":4, "w":1.25},
+        {"x":9.5, "y":4, "w":2.75},
+        {"x":12.25, "y":4, "w":1.5},
+        {"x":16.75, "y":4, "w":1.5}
+      ]
+    },
+    "LAYOUT_all": {
```
Comment (noroadsleft) [inline keyboards/axolstudio/yeti/info.json:77]:
```suggestion
    "LAYOUT_alice_split_bs": {
```

File: keyboards/axolstudio/yeti/info.json [hunk 11]
```diff
@@ -0,0 +1,285 @@
...
+        {"x":1.25, "y":4, "w":1.5},
+        {"x":4.25, "y":4, "w":1.5},
+        {"x":5.75, "y":4, "w":2},
+        {"x":7.75, "y":4, "w":1.25},
+        {"x":9.5, "y":4, "w":2.75},
+        {"x":12.25, "y":4, "w":1.5},
+        {"x":16.75, "y":4, "w":1.5}
+      ]
+    },
+    "LAYOUT_full_rshift": {
```
Comment (noroadsleft) [inline keyboards/axolstudio/yeti/info.json:147]:
```suggestion
    "LAYOUT_alice_full_rshift": {
```

File: keyboards/axolstudio/yeti/info.json [hunk 12]
```diff
@@ -0,0 +1,285 @@
...
+        {"x":1.25, "y":4, "w":1.5},
+        {"x":4.25, "y":4, "w":1.5},
+        {"x":5.75, "y":4, "w":2},
+        {"x":7.75, "y":4, "w":1.25},
+        {"x":9.5, "y":4, "w":2.75},
+        {"x":12.25, "y":4, "w":1.5},
+        {"x":16.75, "y":4, "w":1.5}
+      ]
+    },
+    "LAYOUT_split_bs": {
```
Comment (noroadsleft) [inline keyboards/axolstudio/yeti/info.json:215]:
```suggestion
    "LAYOUT_alice_split_bs_full_rshift": {
```

File: keyboards/axolstudio/yeti/keymaps/default/keymap.c [hunk 13]
```diff
@@ -0,0 +1,34 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    /* Base */
+    [0] = LAYOUT(
```
Comment (noroadsleft) [inline keyboards/axolstudio/yeti/keymaps/default/keymap.c:20]:
```suggestion
    [0] = LAYOUT_alice(
```

File: keyboards/axolstudio/yeti/keymaps/default/keymap.c [hunk 14]
```diff
@@ -0,0 +1,34 @@
...
+        KC_PAUS, KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,           KC_BSPC,
+        KC_PSCR, KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC,          KC_BSLS,
+        KC_F5,   KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,
+                 KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT,            MO(1),
+                 KC_LCTL,          KC_LALT,          KC_SPC,  KC_LGUI,          KC_SPC,           KC_RALT,                                    KC_RCTL
+    ),
+    [1] = LAYOUT(
```
Comment (noroadsleft) [inline keyboards/axolstudio/yeti/keymaps/default/keymap.c:27]:
```suggestion
    [1] = LAYOUT_alice(
```

File: keyboards/axolstudio/yeti/keymaps/via/keymap.c [hunk 15]
```diff
@@ -0,0 +1,48 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    /* Base */
+    [0] = LAYOUT_all(
```
Comment (noroadsleft) [inline keyboards/axolstudio/yeti/keymaps/via/keymap.c:20]:
```suggestion
    [0] = LAYOUT_alice_split_bs(
```

File: keyboards/axolstudio/yeti/keymaps/via/keymap.c [hunk 16]
```diff
@@ -0,0 +1,48 @@
...
+        KC_PAUS, KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC,  KC_DEL,
+        KC_PSCR, KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC,          KC_BSLS,
+        KC_F5,   KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,           KC_ENT,
+                 KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT,            MO(1),
+                 KC_LCTL,          KC_LALT,          KC_SPC,  KC_LGUI,          KC_SPC,           KC_RALT,                                    KC_RCTL
+    ),
+    [1] = LAYOUT_all(
```
Comment (noroadsleft) [inline keyboards/axolstudio/yeti/keymaps/via/keymap.c:27]:
```suggestion
    [1] = LAYOUT_alice_split_bs(
```

File: keyboards/axolstudio/yeti/keymaps/via/keymap.c [hunk 17]
```diff
@@ -0,0 +1,48 @@
...
+        KC_TRNS, KC_GRV,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_HOME, KC_UP,   KC_END,  KC_PGUP, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,
+        KC_TRNS, KC_CAPS, KC_LEFT, KC_DOWN, KC_RGHT, KC_PGDN, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,
+                 KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_VOLD, KC_VOLU, KC_MUTE, KC_MPRV, KC_MNXT, KC_MPLY, KC_TRNS,          KC_TRNS,
+                 KC_TRNS,          KC_TRNS,          KC_TRNS, KC_TRNS,          KC_TRNS,          KC_TRNS,                                     KC_TRNS
+    ),
+    [2] = LAYOUT_all(
```
Comment (noroadsleft) [inline keyboards/axolstudio/yeti/keymaps/via/keymap.c:34]:
```suggestion
    [2] = LAYOUT_alice_split_bs(
```

File: keyboards/axolstudio/yeti/keymaps/via/keymap.c [hunk 18]
```diff
@@ -0,0 +1,48 @@
...
+        KC_TRNS, KC_GRV,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_HOME, KC_UP,   KC_END,  KC_PGUP, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,
+        KC_TRNS, KC_CAPS, KC_LEFT, KC_DOWN, KC_RGHT, KC_PGDN, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,
+                 KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_VOLD, KC_VOLU, KC_MUTE, KC_MPRV, KC_MNXT, KC_MPLY, KC_TRNS,          KC_TRNS,
+                 KC_TRNS,          KC_TRNS,          KC_TRNS, KC_TRNS,          KC_TRNS,          KC_TRNS,                                     KC_TRNS
+    ),
+    [3] = LAYOUT_all(
```
Comment (noroadsleft) [inline keyboards/axolstudio/yeti/keymaps/via/keymap.c:41]:
```suggestion
    [3] = LAYOUT_alice_split_bs(
```

File: keyboards/axolstudio/yeti/rules.mk [hunk 19]
```diff
@@ -0,0 +1,22 @@
...
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
```
Comment (noroadsleft) [inline keyboards/axolstudio/yeti/rules.mk:22]:
```suggestion
AUDIO_ENABLE = no           # Audio output

LAYOUTS = alice alice_split_bs
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]
## Summary:

### Renames the layouts as follows:

`LAYOUT` -> `LAYOUT_alice`
`LAYOUT_all` -> `LAYOUT_alice_split_bs`
`LAYOUT_split_bs` -> `LAYOUT_alice_split_bs_full_rshift`
`LAYOUT_full_rshift` -> `LAYOUT_alice_full_rshift`

### Adds both current Alice community layouts to `rules.mk`

----

GitHub tip: You can apply multiple suggestions to a single commit by using the [Files Changed tab](https://github.com/qmk/qmk_firmware/pull/11051/files).

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**